sbt-plugin/src/main/scala/akka/grpc/sbt/AkkaGrpcPlugin.scala

```diff
@@ -38,13 +38,42 @@ object AkkaGrpcPlugin extends AutoPlugin {
       akkaGrpcCodeGenerators := GeneratorAndSettings(ScalaServerCodeGenerator, akkaGrpcCodeGeneratorSettings.value) :: Nil,
       akkaGrpcModelGenerators := Seq[Target]((JvmGenerator("scala", ScalaPbCodeGenerator), akkaGrpcCodeGeneratorSettings.value) -> sourceManaged.value),
 
+      PB.protoSources in Compile += target.value / "protobuf",
+
+      sourceDirectory in PB.recompile := sourceDirectory.value / "protobuf",
+      resourceDirectory in PB.recompile := sourceDirectory.value / "protobuf",
+
+      target in PB.recompile :=
+        crossTarget.value / "protobuf" / Defaults.nameForSrc(configuration.value.name),
+      // TODO: review managedSources
+      managedSourceDirectories += (target in PB.recompile).value,
+      unmanagedResourceDirectories += (resourceDirectory in PB.recompile).value,
+      watchSources in Defaults.ConfigGlobal ++= (sources in PB.recompile).value,
```

Comment:
This is the key setting.

Reply:
Correction: these three lines are key (first two for dev-mode in play/lagom) and last for regular `~task`'s in `sbt`.